Source/Classes/SLKTextInputbar.m

```diff
@@ -316,15 +316,22 @@ - (BOOL)limitExceeded
 
 - (CGFloat)slk_appropriateRightButtonWidth
 {
-    NSString *title = [self.rightButton titleForState:UIControlStateNormal];
-    CGSize rigthButtonSize = [title sizeWithAttributes:@{NSFontAttributeName: self.rightButton.titleLabel.font}];
-    
     if (self.autoHideRightButton) {
```

Comment:
I also moved this code to the top of the function, since it doesn't depend on `title` or `rightButtonSize `